src/application-development/configuration.md

```diff
@@ -0,0 +1,25 @@
+# Configuration
+
+The [esp-config] crate provides a way to manage configuration settings.
+
+## Usage
+
+The full list of available options can be found in the documentation of the `esp-*` crate you are using. For ESP32-C6's options see [configuration].
+
+For example, in esp-hal if you want to place a function in your code in the RAM (`ESP_HAL_CONFIG_PLACE_SPI_DRIVER_IN_RAM`), which is set to `false` as default, you need to create an environment variable with the same name and modify its value:
+
+```toml
+# .cargo/config.toml
+
+[env]
+ESP_HAL_CONFIG_PLACE_SPI_DRIVER_IN_RAM="true"
+```
+
+After modifying the `.cargo/config.toml` and `[env]` section, **clean build** is recommended.
+
+> ⚠️ **Note**: Setting the `env-vars` on the command line will have precedence over env section.
```

Comment:
Would prefer to use the [Note synax](https://github.com/orgs/community/discussions/16925) here instead.

Reply:
Not sure if that works on mdbook, and if we change it we should update https://github.com/esp-rs/book/blob/main/rust-doc-style-guide.md#admonitions

Reply:
Ahh sorry wasn't thinking, used to things being on GitHub 😅